akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/BehaviorTestKit.scala

```diff
@@ -75,7 +75,7 @@ abstract class BehaviorTestKit[T] {
   /**
    * The self reference of the actor living inside this testkit.
    */
-  def getRef(): ActorRef[T] = selfInbox.getRef()
+  def getRef(): ActorRef[T] = selfInbox().getRef()
```

Comment:
I prefer changing getRef, selfInbox, TypedProps.apply[T], Props.apply[T] to nullary defs. There are many changes in this PR related to those defs, especial the last one.
But if we do that => break source compatibility 😁

Reply:
in the `javadsl` I don't think it matters much, as it breaks source compatibility only for Scala consumers, not for Java ones, right?